samples/java/jre/console/src/com/microsoft/cognitiveservices/speech/samples/console/Main.java

```diff
@@ -122,6 +123,8 @@ public static void main(String[] args) {
                 case "p":
                     SpeechSynthesisSamples.synthesisWordBoundaryEventAsync();
                     break;
+                case "q":
+                    SpeechSynthesisScenarioSamples.synthesisServerScenarioAsync();
```

Comment:
break missing here

Reply:
I checked, and this is the same as in the internal master branch, so it's not a bad merge.
We should file a bug, fix it, and include it in the next update.

If you want, we can fix it directly here in the public repot, so it is fixed before the next release.

---
In reply to: [387453222](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/529#discussion_r387453222) [](ancestors = 387453222)

Reply:
Yes, please fix it in next update